### DISCUSSION TITLE
unaple to apply as a teacher

### DISCUSSION THREAD

**Discussion by fabricelallemand15** [community/community#41032] [GitHub Education]
Hello
I am a French teacher of computer science and i would like to use GitHub
 for education. I tried to apply one time, but the photo of my card was too bad. I wanted to try another time with a better scan, but the system asks me to apply as a student, although i am a teacher.
Could you help me please ?
I join a screenshot
Thanks
![Screenshot_20221207_204643](https://user-images.githubusercontent.com/19834776/206284662-7235cc86-35d3-428c-b1ee-a70526f3c69d.png)

**Comment by emploiyer** [comment]
Ok try to login being a student after you can change it

Thanks

**Comment by fabricelallemand15** [comment]
ok i was accepted as a student ! But i wanted to apply as a teacher (i am actually a teacher)...

**Reply by samus-aran** [reply]
Hi @fabricelallemand15 👋 

Thanks for reaching out! We'll raise this with the team to assist with the incorrect status. We'll be in touch of any updates 👍

But we would very much appreciate your help to understand how you were able to apply as a student when you wished to apply as a teacher. Was there any indication on the application page that failed to indicate that you were submitting as a student rather than faculty?

**Reply by fabricelallemand15** [reply]
No, I checked the “Teacher button”. The problem appeared when I tried to join a new photo of my card, as shown on the screenshot in my first message.

**Reply by samus-aran** [reply]
Thank you for providing more information!

We've revoked your student status so you can proceed with your current application as faculty. We do ask you to re-apply again so the submission can be verified as a teacher. Let us know how you get on!

**Reply by fabricelallemand15** [reply]
I can't give the photo of the card ...
![Screenshot_20221207_225830](https://user-images.githubusercontent.com/19834776/206306851-41b9d746-e290-4e11-9855-d8915e2cb183.png)

**Comment by fabricelallemand15** [comment]
I managed to create an assignment for a test, but as you can see on the sccreenshot i am asked to get verified and I can't
![Screenshot_20221207_214319](https://user-images.githubusercontent.com/19834776/206291614-39cef8e8-69e2-456f-a11a-2d1e23689f47.png)

**Comment by fabricelallemand15** [comment]
It's worrying... I just received a mail which asked me to reverify my academic status.
I tried to do that but ... see the screenshot.
I am going to give up.
![Screenshot_20221207_225830](https://user-images.githubusercontent.com/19834776/206305773-eaf2e385-4ec0-4dda-9b7f-15fe98706110.png)

**Reply by samus-aran** [reply]
Hi @fabricelallemand15 

I can understand your frustration. I hope I can provide some clarity. As teacher and student statuses are not interchangeable we need to revoke one to allow the user to apply for the other. This is why you have received a notice to re-apply.

Regards to the re-use of evidence we kindly ask to re-take a photo of your evidence. The automation verification does pick up on the re-use of the same image file and in this case as you had used it in a previously successful application (the incorrect student status). You can use the same evidence but an alternate photo.

Hope this helps with your application. 👍

**Reply by fabricelallemand15** [reply]
OK i made a new scan to reapply. This time it tells me the name of the school is not readable. Will I manage one time to apply ?

**Reply by samus-aran** [reply]
Hi @fabricelallemand15 

I'm assuming the evidence is in French, it should pick up this language. Another tip is if the name of the school isn't readable perhaps the school you have selected during your application and the school name on the evidence do not correspond. Do they match? If not then I would suggest providing evidence that contains the name of the school you have selected. As you can imagine GitHub processes countless institutions so the application process needs to have at least matching references to school names.

Hope this helps. 👍

**Comment by fabricelallemand15** [comment]
Help please, I'm trying to prove I am a teacher since yesterday ! It's awfull !
